sagemaker_processing/fairness_and_explainability/fairness_and_explainability.ipynb

```diff
@@ -373,13 +368,44 @@
     "                                      s3_output_path=bias_report_output_path,\n",
     "                                      label='Target',\n",
     "                                      headers=training_data.columns.to_list(),\n",
-    "                                      dataset_type='text/csv')\n",
-    "\n",
+    "                                      dataset_type='text/csv')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "A `ModelConfig` object communicates information about your trained model. To avoid additional traffic to your production models, SageMaker Clarify sets up and tears down a dedicated endpoint when processing.\n",
+    "* `instance_type` and `instance_count` specify your preferred instance type and instance count used to run your model on during SageMaker Clarify's processing. The testing dataset is small so a single standard instance is good enough to run this example. If your have a large complex dataset, you may want to use a better instance type to speed up, or add more instances to enable Spark parallelization.\n",
+    "* `accept_type` denotes the endpoint response payload format, and `content_type` denotes the payload format of request to the endpoint."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
     "model_config = clarify.ModelConfig(model_name=model_name,\n",
-    "                                   instance_type='ml.c5.xlarge',\n",
+    "                                   instance_type='ml.m5.xlarge',\n",
     "                                   instance_count=1,\n",
-    "                                   accept_type='text/csv')\n",
-    "\n",
+    "                                   accept_type='text/csv',\n",
+    "                                   content_type='text/csv')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "A `ModelPredictedLabelConfig` provides information on the format of your predictions. XGBoost model outputs probabilities of samples, so SageMaker Clarify invokes the endpoint then uses `probability_threshold` to convert the probability to binary for bias analysis. Prediction above the threshold indicates 1 and below or equal indicates 0."
```

Comment:
* to binary labels
* prediction above the threshold is interpreted as label value '1' and below or equal as  label  value '0' 
* indicates -> indicate

Reply:
Thanks, updated in the new revision.